docs/Commands.md

```diff
@@ -105,18 +105,27 @@ the Security Level.
 This command only has an effect once; the reporter is able to make the ticket public afterwards again.
 
 ## $ARISA_PURGE_ATTACHMENT
-| Entry       | Value                                                      |
-| ----------- | ---------------------------------------------------------- |
-| Syntax      | `$ARISA_PURGE_ATTACHMENT <username> [<min ID>] [<max ID>]` |
-| Permissions | Mod+                                                       |
+| Entry       | Value                                                                   |
+| ----------- | ----------------------------------------------------------------------- |
+| Syntax      | `$ARISA_PURGE_ATTACHMENT [from <minId>] [to <maxId>] [by <username>]` |
+| Permissions | Mod+                                                                    |
```

Comment:
Really minor, but the indentation does not seem to match:
```suggestion
| Entry       | Value                                                                 |
| ----------- | --------------------------------------------------------------------- |
| Syntax      | `$ARISA_PURGE_ATTACHMENT [from <minId>] [to <maxId>] [by <username>]` |
| Permissions | Mod+                                                                  |
```

(sorry forgot to add this comment before creating the review comment ...)